README.md

```diff
@@ -321,6 +321,14 @@ This will generate a `enum Name` in an extension of `UIColor`, with one `case` p
 
 The text file is expected to have one line per color to register, each line being composed by the Name to give to the color, followed by ":", followed by the Hex representation of the color (like `rrggbb` or `rrggbbaa`, optionally prefixed by `#` or `0x`). Whitespaces are ignored.
 
+SwiftGen can also read `*.clr` files. For example you can use this comand to generate colors from one of your system palletes:
```

Comment:
`palletes` -> `palettes`

Would also be interesting to link to Natasha's blog post which explain how to create those color palettes and where they are located in ~/Library`, to encourage people to use them.